.github/workflows/s3-mirror.yml

```diff
@@ -31,6 +31,8 @@ jobs:
       - name: Wait for other instances of this workflow to finish
         # It's not safe to run two S3 sync operations concurrently with different files
         uses: softprops/turnstyle@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           same-branch-only: no
```

Comment:
Not great that we have to give it the token. It's a third-party action so always more risk of it being exfiltrated. Still, this is probably a bit too paranoid, we'll probably be ok.

Still, odd that this is doing enough API requests to hit the limits. I'd only expect that with lots of stuff being merged at the same time. So possible that we're only hitting the limit here because we used it up elsewhere.

Reply:
Should be resolved by reducing the polling frequency of [turnstyle](https://github.com/softprops/turnstyle) github action, the default is 60 seconds: https://github.com/softprops/turnstyle?tab=readme-ov-file#inputs